lib/sassc/functions_handler.rb

```diff
@@ -76,6 +40,21 @@ def setup(native_options)
 
     private
 
+    def arguments_from_native_list(native_argument_list)
+      native_argument_list_length = Native.list_get_length(native_argument_list)
+
+      (0...native_argument_list_length).map do |i|
+        native_value = Native.list_get_value(native_argument_list, i)
+        Script::ValueConversion.from_native(native_value, @options)
+      end.compact
+    end
+
+    def to_native_value(sass_value)
+      sass_value ||= Script::String.new("") # null response
```

Comment:
I feel like if we’re going to have a comment here, it should explain more.

Alternatively, we could have a `Script::Nil` type that’s always just a `Script::String.new("")`